skfem/assembly/basis/interior_basis.py

```diff
@@ -130,9 +131,9 @@ def refinterp(self,
 
         return M, w.flatten()
 
-    def probes(self, x: np.ndarray) -> Callable[[np.ndarray], np.ndarray]:
-        """Return a function handle, which can be used for finding
-        the values on points `x` of a given solution vector."""
+    def probes_data(self, x: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
Rename to `_probes_data`?

Reply:
Another comment: I've been importing `ndarray` explicitly from numpy. No any real reason but to make the type definitions shorter (they are sometimes pretty long). You can also here write them without `np` because `ndarray` is already imported.

Reply:
> Rename to `_probes_data`?

Yes, of course; done.

I was also wondering whether we wanted to rethink `InteriorBasis.probes`; whether we need or want both the function and matrix versions.  `.probes` is pretty new, indeed unreleased, so I think it could be switched to return the matrix instead of the function and the vestigial `InteriorBasis.interpolator` still reimplemented, as is, in terms of the matrix.  What do you think?  I might try that.  Would there be any efficiency difference between the matrix and the function?  I'm thinking it unlikely (but might well find out in #600).

Reply:
In 4e3afb4, I got rid of `_probes_data` anyway; should anyone want the old `phis` and `indices` (though I can't think why) they could be recovered from the `data` and `col` attributes of the `coo_matrix` returned by the revised `probes` method.